### ISSUE TITLE
Always raise 'FutureWaring' when using RandomSurvivalForest

### ISSUE THREAD

**Issue by Ruikun-Li** [sebp/scikit-survival#135]
**Describe the bug**

When using RandomSurvivalForest to predict, there always rasing a 'FutureWaring' that: predict_cumulative_hazard_function will return an array of StepFunction instances in 0.14. Use return_array=True to keep the old behavior. FutureWarning). 
This warning comes from sksurv/tree/tree.py.

**Code Sample to Reproduce the Bug**

```python
X, y = load_whas500()
X = X.astype(float)
estimator = RandomSurvivalForest().fit(X, y)
score = estimator.predict(X)
```

**Expected Results**
In 390 line in sksurv/tree/tree.py, the code should be:
chf = self.predict_cumulative_hazard_function(X, check_input, return_array=True)
instead of the original code:
chf = self.predict_cumulative_hazard_function(X, check_input)

**Actual Results**
No further warning

**Versions**
Please execute the following snippet and paste the output below.

```python
System:
    python: 3.6.9 |Anaconda, Inc.| (default, Jul 30 2019, 14:00:49) [MSC v.1915 64 bit (AMD64)]
executable: C:\Users\lrk19\Anaconda3\envs\dl-pytorch\python.exe
   machine: Windows-10-10.0.18362-SP0

Python dependencies:
          pip: 19.3.1
   setuptools: 42.0.2.post20191203
      sklearn: 0.23.1
        numpy: 1.17.4
        scipy: 1.4.1
       Cython: 0.29.21
       pandas: 0.25.3
   matplotlib: 3.1.2
       joblib: 0.14.1
threadpoolctl: 2.1.0

Built with OpenMP: True
sksurv: 0.13.0
cvxopt: 1.2.3
cvxpy: 1.0.15
numexpr: 2.7.1
osqp: 0.5.0
```

**Comment by sebp** [comment]
It should fixed in 0.13.1. Could you please try with 0.13.1?

**Comment by sebp** [comment]
I'm closing this issue as no further information has been provided. Please feel free to reopen this issue if you can provide the requested information.

**Comment by Ruikun-Li** [comment]
> It should fixed in 0.13.1. Could you please try with 0.13.1?

I have checked it, and it was actually fixed in 0.13.1!
Thanks for your reply and please forgive me for my slow response.
